### PR TITLE
5560 Hide community spotlight setting if not enabled

### PR DESCRIPTION
**Pull request by sakshi-keepworks**

### CODE REVIEW

File: app/views/aspects/_no_contacts_message.haml [hunk 1]
```diff
@@ -7,5 +7,6 @@
   %br
   %br
   = t('.try_adding_some_more_contacts')
-  != t('.or_spotlight', :link => link_to(t(".community_spotlight") , community_spotlight_path))
+  - if current_user.show_community_spotlight_in_stream?
```
Comment (Flaburgan) [inline app/views/aspects/_no_contacts_message.haml]: This is not the good condition, we want to hide this message if the podmin didn't activate community spotlight in diaspora.yml. Besides, #5560 is about the setting in the user panel, not this message, but you are right, both have to be hidden.

**Comment by svbergerem** [comment]
You are now hiding the information that one could also add contacts from the community spotlight if someone doesn't want to see the community spotlight in its stream? I don't think that was the behavior #5560 was referring to.

This message and the setting itself should be hidden if the community spotlight are disabled for the whole pod. (See https://github.com/diaspora/diaspora/blob/develop/app/views/contacts/_sidebar.html.haml#L4)

**Comment by Flaburgan** [comment]
@svbergerem I was first! :p

**Comment by svbergerem** [comment]
@sjain1107 Btw. it is great to see you contributing again. :)

**Comment by sjain1107** [comment]
Thanks :) I'm happy that atleast something is correct ;)

**Comment by sjain1107** [comment]
Question: Where are the settings and how to enable or disable community spotlight for the whole pod?

**Comment by svbergerem** [comment]
Settings:
https://github.com/diaspora/diaspora/blob/develop/app/views/users/edit.html.haml#L88
(header dropdown -> settings -> stream settings)

Enable/Disable the community spotlight for the pod:
https://github.com/diaspora/diaspora/blob/develop/config/diaspora.yml.example#L430

**Comment by Flaburgan** [comment]
So the test you want to write is `- if AppConfig.settings.community_spotlight.enable?`

**Comment by goobertron** [comment]
Wow, that was quick! Thanks for your PR, Sakshi, and it's good to see you contributing again.

**Comment by sjain1107** [comment]
@svbergerem @Flaburgan @goobertron Thanks! Will look into it.

**Comment by jhass** [comment]
Two simple controller tests that assert the checkbox is included in the response or not depending on the setting would be nice :)

**Comment by sjain1107** [comment]
@jhass Added tests. Please review. :)

**Comment by jhass** [comment]
Great, thank you!

**Pull request merged**